docs/contracts.rst

```diff
@@ -194,10 +194,10 @@ In the following example, ``D``, can call ``c.getData()`` to retrieve the value
     contract C {
         uint private data;
 
-        function f(uint a) private returns(uint b) { return a + 1; }
+        function f(uint a) private pure returns(uint b) { return a + 1; }
         function setData(uint a) public { data = a; }
         function getData() public returns(uint) { return data; }
```

Comment:
This should be `view` if we are upgrading this non-compiling example :)

Reply:
Oh yeah, that's true... 😭

Reply:
Also, can you help me understand the build failure message? I'm reading the output but I don't see the error. It looks like it's some sort of file-not-found within the build machine.

Reply:
You haven't changed `getData()` and it is lacking a visibility specifier, but should be `view`. The others you have changed to `pure` makes sense.

I am not sure about the error message unfortunately.

Reply:
Ah I see. I can update `getData` as well.

Reply:
@kronosapiens the error message you are getting is a known issue.